generator/main.go

```diff
@@ -273,15 +337,182 @@ func generateDoc(destination string) (interface{}, error) {
 	}
 
 	if destination == "Rockset" {
-		doc["_id"] = guuid.New().String()
+		// Monotonic ids
+		doc["_id"] = formatDocId(max_doc_id)
+		max_doc_id = max_doc_id + 1
 	}
 
-	doc["_event_time"] = getCurrentTimeMicros()
+	// Use _ts instead of _event_time as _ts is mutable
+	doc["_ts"] = getCurrentTimeMicros()
```

Comment:
can we also keep adding `_event_time` too for inserts?

Reply:
It does prevent the document from being overwritten I believe as _event_time is immutable. I'm fine with adding it back 🤷‍♂️

Reply:
we can just add it for insert operations?

Reply:
It can only be added for insert operations otherwise the whole patch will fail. [JSONPatch operations are atomic](https://en.wikipedia.org/wiki/JSON_Patch), so trying to set an immutable value will cause the patch to be a no-op. Our API fails to actually report if a patch failed because it is async...